strings/strings.json

```diff
@@ -7596,5 +7596,9 @@
   "Send": {
     "en": "Send",
     "fr": "Envoyer"
+  },
+  "UsefulnessFeedbackReason": {
+    "en": "What's wrong?",
```

Comment:
I'd go with:

Reason / Raison

as a legend

Reply:
Perfect, I'll change it. Thank you!